core/src/main/web/app/utils/basic/commonui.js

```diff
@@ -83,13 +83,15 @@
 
         var showDropdown = function() {
           var togglePosition = toggle.offset();
+          var $notebook = $('bk-notebook');
+          var notebookPosition = $notebook.offset();
```

Comment:
this method is now going to cause two layout calcs .. causing some perf hits :| perhaps we could use RAQ and do the majority of the reflow work on the next available paint frame (this change would require some refactoring)